runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java

```diff
@@ -192,6 +196,9 @@ private void onDeselect()
 	@Subscribe
 	private void onMenuOptionClicked(MenuOptionClicked ev)
 	{
+		//check to see if its a skill menu
+		boolean isSkill = ev.getMenuOption().startsWith("View");
```

Comment:
This is probably better placed in the if (ev.getMenuAction() == MenuAction.RUNELITE) block where it's used. It's getting set everytime this method is called here even for unrelated MenuOptions